test/addons_test.go

```diff
@@ -341,5 +349,72 @@ configInline:
     protocol: layer2
     addresses:
     - "172.17.1.200-172.17.1.250"
+`,
+	// Copied from addons/6.8.x/elasticsearch-5.yaml and edited to lower resource
+	// limits so it can deploy on a kind cluster.
+	// Added and edited lines are marked with "# override".
+	"elasticsearch": `
```

Comment:
I copied the complete addon config because I couldn't find a convenient way to retrieve it and merge it with the overrides :\

Reply:
:thinking: you should be able to read the entire local file in and utilize that, rather than needing to place overrides here?

Maybe I'm misunderstanding something?

Reply:
I'm still wondering about this :thinking:

Reply:
Oops I missed this comment! I assume we don't want to special-case how the logging addons get their overrides applied (Prometheus addons will need similar overrides as well), so I think I'll have to figure out how to update `overrides()` to patch values rather than overwrite them entirely, without breaking the metallb override. I'll take a stab at it.

Reply:
:+1:

Reply:
https://github.com/mesosphere/kubernetes-base-addons/pull/208/commits/429178610a01edf4c1fc5a3560ef4fcfd56bb802 I took a look at the addon while the test was running to confirm that the overrides were merged into its values.